pkgs/applications/networking/cluster/argocd/default.nix

```diff
@@ -28,13 +28,15 @@ buildGoModule rec {
       "-s" "-w"
       "-X ${package_url}.version=${version}"
       "-X ${package_url}.buildDate=unknown"
-      "-X ${package_url}.gitCommit=${commit}"
-      "-X ${package_url}.gitTag=${tag}"
+      "-X ${package_url}.gitCommit=unknown"
```

Comment:
Is this recommend upstream? Otherwise I would prefer that you kept the old way.

Reply:
Nope, you either remove it/do not set value or set the correct value, I manually set this value to prevent any weird issue.

Reply:
https://github.com/argoproj/argo-cd/blob/master/Makefile#L133

Reply:
https://github.com/argoproj/argo-cd/blob/b37eee1054e42c873699460dd5e2447c2f9fe5a6/common/version.go#L10

Reply:
Then I think it is better that you keep the commit to be specified like before (and add maybe a comment remembering people that they need to update the commit if they update the version).

Reply:
Why? I mean is one more thing to keep control and ryantm's bot does not update by default, also I think these strategies of tracking is a bit irrelevant in Nix environment (but ok outside it).

Reply:
I take similar approach in my other packages, I know that is a bit "ugly" not having it in version output but well, I do not know if that is worth the "hassle".

Reply:
Setting versioning attributes to random values is known to cause some strange issues sometimes, because you never know if people are using this do to some validation. A good example: https://github.com/NixOS/nixpkgs/pull/161801.

> I mean is one more thing to keep control and ryantm's bot does not update by default

You can always write a update script and solve those two issues.

Reply:
Well, then I will do this time manually and work on that.